fbchat/models.py

```diff
@@ -436,6 +439,87 @@ def __repr__(self):
     def __unicode__(self):
         return '<Mention {}: offset={} length={}>'.format(self.thread_id, self.offset, self.length)
 
+class Poll(object):
+    #: ID of the poll
+    uid = None
+    #: Title of the poll
+    title = None
+    #: List of :class:`PollOption`
+    options = None
+    #: Options count
+    options_count = None
+
+    def __init__(self, title, options):
+        """Represents a poll"""
+        self.title = title
+        self.options = options
+    
+    def __repr__(self):
+        return self.__unicode__()
+
+    def __unicode__(self):
+        return '<Poll ({}): {} options={}>'.format(self.uid, repr(self.title), self.options)
+
+class PollOption(object):
+    #: ID of the poll option
+    uid = None
+    #: Text of the poll option
+    text = None
+    #: Whether vote when creating or client voted
+    vote = None
+    #: ID of the users who voted for this poll option
+    voters = None
+    #: Votes count
+    votes_count = None
+
+    def __init__(self, text, vote=False):
+        """Represents a poll option"""
+        self.text = text
+        self.vote = vote
+    
+    def __repr__(self):
+        return self.__unicode__()
+
+    def __unicode__(self):
+        return '<PollOption ({}): {} voters={}>'.format(self.uid, repr(self.text), self.voters)
+
+class Event(object):
```

Comment:
I think we should use the name `Plan` instead of `Event` throughout the code, since that's what Facebook calls them, and since the word "event" could be interpreted as a lot of different things, while "plan" is more specific

Reply:
Done